README.md

```diff
@@ -1,76 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Catalog of my things
 
-> One paragraph statement about the project.
+> This app helps users to keep a record of different types of things they own
 
-![screenshot](./app_screenshot.png)
+This is a simple console app that allows user to manage collections of books they own
 
-Additional description about the project and its features.
+This app can do the following:
 
-## Built With
+- Show a list of books
+- Show a list of labels
+- Store books
 
-- Major languages
-- Frameworks
-- Technologies used
+All the books and labels are stored in JSON files
 
-## Live Demo
+## Built With
 
-[Live Demo Link](https://livedemo.com)
+- Ruby
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
+- Ruby >= 3.0.1
+
 ### Setup
 
-### Install
+- Run 'git clone https://github.com/LOctavio/catalog-of-my-things.git' to get a local copy
 
 ### Usage
 
-### Run tests
+- Run 'ruby main.rb' to run the project 
```

Comment:
- [ ] Your project looks amazing there and it works great 🥇 However I would like to suggest you try to make the Setup/Usage part of your application a little more user friendly. try to make it clear enough that someone with minimal coding experience can install and use your application.